source/support.tex

```diff
@@ -3378,7 +3378,7 @@
 \tcode{parmN}.}
 If the parameter
 \tcode{parmN}
-is of a reference type, or of a type that is not compatible with the
+is of a reference type, or of a type that is not the same as the
```

Comment:
@zygoloid: The signed/unsigned relaxation applies to the arguments passed to the ellipsis. By contrast, the text in question applies to the argument for the *named* parameter `paramN`.